src/com/esotericsoftware/kryo/serializers/FieldSerializer.java

```diff
@@ -602,6 +604,12 @@ public void removeField (CachedField removeField) {
 		return fields;
 	}
 
+    /** Get all transient fields controlled by this FieldSerializer
+     * @return all transient  fields controlled by this FieldSerializer */
+	public CachedField[] getTransientFields () {

```

Comment:
It is for custom serializer.
In case of custom serializer, there is no way to get transient fields so that it is hard to serialize transient fields.
I think that making `fields` and `transientFields` from private to protected could be good alternative if you decide to do not add more public api.